modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js

```diff
@@ -89,6 +89,13 @@ const ObjectField = ({
 				else if (relationshipType || system) {
 					return false;
 				}
+				else if (
+					(focusedFieldType === 'document_library' ||
+						focusedFieldType === 'image') &&
+					type === 'Long'
```

Comment:
Hi, @rodrigopaulino. 
The reason for the file extensions being hardcoded is that the initial idea of this mapping is to perform a quick win because of the urgency that our client needs this mapping done. Later on, if necessary, this can be improved, but for the current scenario, this version meets our use case.

For this particular piece of code, the logic was done this way because the object has declared the type for its fields as long only for fields that store the id of an attachment, saving the id of the file in the documents and media.

So I followed the method pattern using the compare with type, changing the compare parameter could make the method construction inconsistent.